### ISSUE TITLE
MunkiAdmin fails to run makecatalogs

### ISSUE THREAD

**Issue by WorkerB** [hjuutilainen/munkiadmin#125]
I have recently installed MunkiAdmin on a new computer and after installing munkitools 2.7 and MunkiAdmin 1.4.2, I mounted our munki_repo and then set to work updating a few packages. However, when I tried to save, I was presented with the error in MunkiAdmin of "munkitools error - can't find makecatalogs" (attached). 

I have run _sudo /usr/local/munki/munkiimport --configure_ to set the repo location and I have also manually run _sudo /usr/local/munki/makecatalogs_ and this works fine. I have also checked the location of makecatalogs in the MunkiAdmin preferences, and this is correct. However, I have noticed that MunkiAdmin is not able to determine the version of munkitools installed, which would indicate there is a problem with MunkiAdmin recognising munkitools. I have tried uninstalling and re-installing munkitools (with reboots in each case), as well as uninstalling and re-installing MunkiAdmin to no avail.

Everything seems to be working fine with munkitools, including Managed Software Update - except that MunkiAdmin refuses to see makecatalogs or run it. 

![munkiadmin error](https://cloud.githubusercontent.com/assets/12911279/15489568/16c4b990-21a4-11e6-83b7-f073223295d8.png)

![munkiadmin munkitools not found](https://cloud.githubusercontent.com/assets/12911279/15493185/973b9b5e-21c5-11e6-8c98-f3a43bcf869a.png)

**Comment by hjuutilainen** [comment]
Thanks for reporting.

Could you run the commands `/usr/local/munki/makecatalogs --version` and `ls -la /usr/local/munki/` and post the output here? Run them as the same user as you're running MunkiAdmin (without `sudo`).

**Comment by WorkerB** [comment]
I have run the two commands and the result is "permission denied". I am logged in as a local admin user. Running 10.10.5 on a newly installed laptop. I first tried the same setup on 10.11.2 with the same result and thought it might be El Capitan so reinstalled with Yosemite.

admin$ /usr/local/munki/makecatalogs --version
-bash: /usr/local/munki/makecatalogs: Permission denied

admin$ ls -la /usr/local/munki/
ls: : Permission denied

**Comment by hjuutilainen** [comment]
The permissions are incorrect on your /usr/local/munki directory (or on /usr/local directory). Run the following to try to fix the munki part:

```
sudo chown -R root:wheel /usr/local/munki/
sudo chmod -R ugo+r /usr/local/munki/
```

Then run the earlier commands again to see if allowing read for everyone was enough.

**Comment by WorkerB** [comment]
The chown wasn't necessary as root:wheel was already the owner of /usr/local/munki. I was able to fix the problem with:

`sudo chmod -R ugo+rx /usr/local/munki/ `

I've identified the cause of this permissions error to be due to an installation order error (in my thin image setup) with a couple of scripts setting the munki repo and other configurations **before** the munkitools installer had run. So, the  /usr/local/munki directory was being created by the script and not by the munkitools installer resulting in the incorrect file permissions.

I really appreciate your assistance even though the issue was not due to a bug or actual issue in MunkiAdmin. Keep up the great work!
